modules/installation/pages/pubcloud-setup.adoc

```diff
@@ -1,20 +1,23 @@
 [[public-cloud-setup]]
 = Public Cloud Setup
 
-Public Cloud providers pre-install {productname}, so you do not need to perform any installation steps.
-However, you must have the correct {productname} and {sle} entitlements to receive updates before you can log in.
+{productname} is pre-installed in images available in the Public Cloud. You do not need to perform any installation steps.
+However, you must have the correct {productname} and {sle} entitlements to receive updates before you can manage client systems.
 
-For most bring your own service (BYOS) images, these entitlements are provided by {suse}, and can be obtained using the {scc}.
-If you are using an on demand image, you need to do some extra preparation before you can register to {scc}.
-
-When you have registered, all {sle} modules are activated.
-You can then activate the public cloud module.
+{productname} is only offered as bring your own subscription (BYOS) images. Entitlements are provided by {suse} or a partner, and can be obtained using the {scc}.
+{sle} system images for clients are generally available in 2 flavors, on-demand images (also referred to as pay as you go - PAYG) and BYOS images. If you are managing on-demand instances the entitlement for the client itself is part of the on-demand pricing. For on-demand instances you need to do some extra preparation before you can manage the system with {productname}.
```

Comment:
```suggestion
{sle} system images for clients are generally available as on-demand or pay as you go (PAYG) images, or as bring your own subscription (BYOS) images. 
If you are managing on-demand instances, the entitlement for the client is part of the on-demand pricing. 
For on-demand instances you need to do some extra preparation before you can manage the system with {productname}.
```

Reply:
We probably need to check this entire chapter for hyphenation consistency in "on-demand".